presto-native-sidecar-plugin/src/main/java/com/facebook/presto/sidecar/sessionpropertyproviders/NativeSystemSessionPropertyProvider.java

```diff
@@ -107,4 +132,15 @@ else if (type == TINYINT) {
         }
         return propertyMetadata;
     }
+
+    private URI getSidecarLocation()
+    {
+        Node sidecarNode = nodeManager.getSidecarNode();
+        return HttpUriBuilder.uriBuilder()
+                .scheme("http") // The sidecar is presumed to be colocated with the coordinator
```

Comment:
what about this assumes the sidecar is colocated? I thought the earlier commit added support for the sidecar to be any worker.

Reply:
Yes, the comment can be removed.